regulations/static/regulations/js/collapse.js

```diff
@@ -0,0 +1,14 @@
+function makeCollapsable(el) {
+    const collapse_buttons = el.querySelectorAll('.js-collapse');
+    for (const collapse_button of collapse_buttons) {
+        collapse_button.addEventListener('click', function() {
+            el.classList.toggle("collapsed");
+        });
+    }
+}
+
+const collapsables = document.querySelectorAll(".js-collapsable")
```

Comment:
How can we show a clean relationship between this class and the functions making it collapsible?

Reply:
This could also be modeled as a statefull element. With buttons that change it's state (collapsed, expanded, etc.)